docs/draft-release-notes.md

```diff
@@ -13,13 +13,14 @@
 - Beta tag: all service offerings tagged as beta and will not be displayed by default in the marketplace. Set the environment variable `GSB_COMPATIBILITY_ENABLE_BETA_SERVICES` to true to enable them. 
 - PostgreSQL: when creating a binding, by default the PostgreSQL connection will be secured via the "verify-full" PosgreSQL configuration. This will require the AWS certificate bundle to be installed, or it can be disabled by setting "use_tls=false"
 - PostgreSQL: a new "provider_verify_certificate" property allows for the PostgreSQL Terraform provider to skip the verification of the server certificate.
-- PostgreSQL: server can reject non-SSL connections by default. Renamed "use_tls" to "require_ssl". Wheh the "require_ssl" property is true, it will make the server require SSL connections. When false (default), the server will accept SSL and non-SSL connections.
+- PostgreSQL: server can reject non-SSL connections by default. Renamed "use_tls" to "require_ssl". When the "require_ssl" property is true, it will make the server require SSL connections. When false (default), the server will accept SSL and non-SSL connections.
 - PostgreSQL: Enhanced Monitoring. Amazon RDS provides metrics in real time for the operating system (OS) of the DB instance. Enhanced Monitoring enables all the system metrics and process information for the RDS DB instances on the console.
 - PostgreSQL: Only "instance_class" are now exposed when provisioning or updating an instance. The previous “cores” abstraction is deprecated, in favour of using the underlying AWS instance class property.
 - PostgreSQL: Automated backups can now be scheduled through "backup_window". By default, the automated backups are disabled.
 - PostgreSQL: Automated backups can be customised through the following properties: "delete_automated_backups" - delete backups when deleting the instance, defaults to true; "copy_tags_to_snapshot" - copy all instance tags to snapshots, defaults to true. 
 - PostgreSQL: Enable encryption with a custom key. Amazon RDS encrypted DB instances provide an additional layer of data protection by securing data from unauthorized access to the underlying storage. Amazon RDS uses an AWS KMS key to encrypt these resources, and now a custom key with the desired configuration can be used.
-- PostgreSQL: Added deprecation warning to `cores` property and made it optional. It is recomended to use the `instance_class` property instead. 
+- PostgreSQL: Added deprecation warning to `cores` property and made it optional. It is recommended to use the `instance_class` property instead. 
+- PostgreSQL: Monitoring DB load with Performance Insights. Performance Insights expands on existing Amazon RDS monitoring features to illustrate and help you analyze your database performance. With the Performance Insights dashboard, you can visualize the database load on your Amazon RDS DB instance load and filter the load by waits, SQL statements, hosts, or users.
```

Comment:
This belongs in the AWS docs. What we want to get here is a couple of lines explaining what changed in the brokerpak. e.g

PostgreSQL: performance insights can now be enabled and a kms key can be provided to encrypt the performance insights data. Performance insights is disabled by default.

Reply:
Thank you!! I'll change it now